notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -170,7 +170,7 @@
       "source": [
```

Comment:
<p>In addition to transformation -&gt; In addition to transformations</p><p> </p><p>drop the "will" in To enable it, you will need to set <code>run_feature_selection</code> to True. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1378/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>